website/docs/plugins/ray_launcher.md

```diff
@@ -144,12 +144,59 @@ $ python my_app.py --multirun task=1,2,3
 </details>
 
 
-<details><summary>Upload & Download from remote cluster</summary>
+##### Overriding default `ray_aws` config
```

Comment:
I think we need generic explanation page to explain how to configure plugins in general in Hydra 1.1 (that all plugins can refer to from their docs).
It doesn't make sense for each plugin doc page to have to explain the same thing.

What do you think?

Reply:
Makes sense. I will create a new page for configuring plugins.